core/lib/src/test/java/org/opennms/core/tracker/RequestTrackerTest.java

```diff
@@ -366,8 +370,10 @@ public void testTimeoutNoRetries() throws Exception {
 
         long elapsedTime = cb.timeoutTimestamp - req.getSentTimestamp();
 
+        System.out.println("testTimeoutNoRetries processing took " + elapsedTime);
```

Comment:
I would change here to use LOG.info() instead of any System.out.print

Reply:
Updated and force pushed.